src/Configuration.cpp

```diff
@@ -10,12 +10,20 @@
 #include "cpptoml/cpptoml.h"
 #include <fstream>
 #include <unordered_map>
+#include <cstdlib>
 using namespace chemfiles;
 
-const char PATH[] = "~/.chemfilesrc";
+static std::string find_file() {
+    std::string PATH;
+    if (const char* env_home = std::getenv("HOME")) {
```

Comment:
This will only work on Linux. Windows does not define `HOME`, and OS X may not. See this for a possible resolution: http://stackoverflow.com/a/3733955/4692076

But cross-platform path handling is hard =/